index.js

```diff
@@ -415,6 +439,7 @@ module.exports = {
         pull.drain(
           (groupInfo) => {
             foundInvite = true
+            // TODO: loop over all the readKeys we've found and add them all
```

Comment:
todo

Reply:
done. note that the "bob managed to decrypt alice's second message" test was failing before i changed this, even with a decent timeout, which means epochs weren't rediscovered by start() (i think?)